src/app/core/header/notifications/notification-item/notification-item.component.ts

```diff
@@ -10,8 +11,15 @@ export class NotificationItemComponent implements OnInit {
   @Input() notification: Notification;
 
   progressbarMode = 'query';
+  urlAddress: String;
 
   constructor() {}
 
-  ngOnInit() {}
+  ngOnInit() {
+    if (environment.envName === 'dev') {
+      this.urlAddress = 'https://ropsten.etherscan.io/tx/';
```

Comment:
We have this URL loaded on the network environment

https://github.com/ripio/rcn-loans/blob/e435122bf32f230b767d1f071354ee7126a80263/src/environments/environment.prod.ts#L32-L40

You can see an example on the lend button behavior

https://github.com/ripio/rcn-loans/blob/e435122bf32f230b767d1f071354ee7126a80263/src/app/shared/lend-button/lend-button.component.ts#L201